config/examples/Mks/Robin/Configuration.h

```diff
@@ -2062,8 +2062,11 @@
 //
 // ADS7843/XPT2046 ADC Touchscreen such as ILI9341 2.8
 //
-//#define TOUCH_BUTTONS
+#define TOUCH_BUTTONS
```

Comment:
Is the MKS Robin (a board, not a printer) guaranteed to have the touch display?

Reply:
yes the FSMC touchscreen was named TFT ROBIN :p

Reply:
then i dunno, generic boards are "generic"... but well they often sell it with the touchscreen

Reply:
Confirm, MKS Robin and MKS Robin Mini have Touch Screen installed cause the screen comes in one package with the board.